docs/actions.md

```diff
@@ -636,11 +636,13 @@ When you create an OpenWhisk Swift action with a Swift source file, it has to be
 To avoid the cold-start delay, you can compile your Swift file into a binary and then upload to OpenWhisk in a zip file. As you need the OpenWhisk scaffolding, the easiest way to create the binary is to build it within the same environment as it will be run in. These are the steps:
 
 - Run an interactive Swift action container.
-```
-docker run --rm -it -v "$(pwd):/owexec" openwhisk/swift3action bash
-```
-   This puts you in a bash shell within the Docker container. Execute the following commands within it:
+  ```
+  docker run --rm -it -v "$(pwd):/owexec" openwhisk/swift3action bash
+  ```
+
+  This puts you in a bash shell within the Docker container. 
 
+  Execute the following commands within it:
 - Install zip for convenience, to package the binary
```

Comment:
While not directly related to this PR, zip is installed by default in the action, so we can remove this bullet point.